common/test/acceptance/tests/lms/test_teams.py

```diff
@@ -103,6 +110,82 @@ def test_teams_enabled_global_staff(self):
         And the correct content should be on the page
         """
         self.set_team_configuration(
-            {u"max_team_size": 10, u"topics": [self.test_topic]}, enroll_in_course=False, global_staff=True
+            {u"max_team_size": 10, u"topics": self.create_topics(1)},
+            enroll_in_course=False,
+            global_staff=True
         )
         self.verify_teams_present(True)
+
+    def test_list_topics(self):
+        """
+        Scenario: a list of topics should be visible in the "Browse" tab
+        Given I am enrolled in a course with team configuration and topics
+        When I visit the Teams page
+        And I browse topics
+        Then I should see a list of topics for the course
+        """
+        self.set_team_configuration({u"max_team_size": 10, u"topics": self.create_topics(2)})
+        self.teams_page.visit()
+        self.teams_page.browse_topics()
+        self.assertEqual(len(self.teams_page.get_topic_cards()), 2)
```

Comment:
We should also verify that the contents of the cards are what we expect. I suggest creating a helper method which verifies both number of cards as well as the contents of those cards.